### ISSUE TITLE
Unable to export data from Sketch Engine to Lexonomy

### ISSUE THREAD

**Issue by axme100** [elexis-eu/lexonomy#10]
Hello. First of all, thank you for creating this resource! I cannot find detailed documentation for exporting data from Sketch Engine to Lexonomy. If this exists, I would be very happy if you could point me to this information.

Here are the steps I took.
1. Created Lexonomy account
2. Put Lexonomy API key into sketch Engine account
3. Created a Corpus form the Web in Sketch Engine.
4. Selected the 1-Click Dictionary Feature
5. Clicked the Go Button.
6. Click Open.

Upon completing step 6, I am taken to the dictionary but it is blank, there are no headwords or nothing. I have tried to do this with several different corpora and with no success.  Here is a screenshot of the screen says after step 6:

![image](https://user-images.githubusercontent.com/8117193/46757421-b68ec680-cc8f-11e8-963f-ac986b3b2fc6.png)

When clicking on this dictionary you can see that no data has been imported:

![image](https://user-images.githubusercontent.com/8117193/46757498-f2c22700-cc8f-11e8-975c-3fe277b455ba.png)

**Comment by michmech** [comment]
Please hold while I investigate this!

**Comment by iiegn** [comment]
(maybe i'm slightly hijacking this issue, but) i tried to use the OCD feature in SkE and it returned even less:
 * neither a blank dictionary in lexonomy
 * nor an email that the processed failed

last time i tried (about two months ago) i got an email that 'something had failed' - which, at least, alerted me about some problem. currently, i don't seem to be getting feedback after initiating a SkE->OCD run.

**Comment by iiegn** [comment]
update: just about now
 - i have received an **Automatic notification from Sketch Engine**
 - and **a dictionary has been created** (...with the rather small default of 10 entries... but still!)

**Comment by iiegn** [comment]
i'm closing this issue: an initial version of `SkE -- push data --> lexonomy.eu` is working (again); any further - or new - problem should go into a new issue.

(@axme100 pls. reopen if this - as described above - is still a problem)
